src/main/webapp/i18n/de/metis.json

```diff
@@ -165,6 +166,15 @@
                 "tutor": "Tutor:in: Verantwortlich für die Unterstützung der Studierenden bei ihren Fragen zum Kurs und zu den Übungen.",
                 "student": "Studierende:r: Nimmt an Kursen und Übungen teil und stellt Fragen.",
                 "deleted": "Gelöscht: Nutzer:in wurde gelöscht."
+            },
+            "forward": {
+                "forwardMessage": "Diese Nachricht weiterleiten",
+                "addMessage": "Eine Nachricht hinzufügen",
+                "searchBar": "Nach Name oder Kanal suchen",
+                "viewConversation": "Unterhaltung anzeigen",
+                "forwardedFrom": "Forwarded from",
```

Comment:
Are you intentionally using a english string? Should be  `"forwardedFrom": "Weitergeleitet von",`

Reply:
thanks for notifying this :)